core/unix/rseq_linux.c

```diff
@@ -99,7 +99,14 @@ typedef struct _rseq_region_t {
     bool reg_written[DR_NUM_GPR_REGS];
 } rseq_region_t;
 
-/* We need to store a struct rseq_cs per fragment_t.  To avoid the cost of adding a
+/* We need to store potentially multiple rseq_cs per fragment. */
```

Comment:
Maybe comment that the multiple rseq_cs is because of drbbdup, and not because of the app.

Reply:
Done.